src/containers/sign-on/store/sagas.js

```diff
@@ -213,25 +212,30 @@ function* validateHandle(action) {
   }
 }
 
-function* validateEmail(action) {
+function* checkEmail(action) {
+  if (!isValidEmailString(action.email)) {
+    yield put(signOnActions.validateEmailFailed('characters'))
+    return
+  }
   try {
-    if (!isValidEmailString(action.email)) {
-      yield put(signOnActions.validateEmailFailed('characters'))
-      return
+    const inUse = yield call(AudiusBackend.emailInUse, action.email)
+    if (inUse) {
```

Comment:
maybe worth commenting here on the expected UX around what happens if the email is taken (goes to sign in)